src/streaming/extensions/ProtectionExtensions.js

```diff
@@ -227,9 +231,9 @@ MediaPlayer.dependencies.ProtectionExtensions.prototype = {
      */
     getLicenseServer: function(keySystem, protData, messageType) {
 
-        // Our default server implementations do not do anything with "license-release" or
-        // "individualization-request" messages, so we just send a success event
-        if (messageType === "license-release" || messageType == "individualization-request") {
+        // Our default server implementations do not do anything with "license-release"
+        // messages, so we just send a success event
+        if (messageType === "license-release") {
             return null;
```

Comment:
@bwidtmann I'd like to come up with a better way to do this.  My initial implementation was just a punt, since I didn't want to deal with it at the time.  However, you have introduced a license server that supports "individualization-request" and I have since added support for persistent licenses in ChromeOS, which sends the "license-release" message now.  So, I think we need to move this functionality into the license-server-specific code now.